Assets/Scripts/Game Controllers/WinConditionController.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class WinConditionController : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D collision)
+    {   
+        Debug.Log(collision.name);
+        if(collision.name == "Player")
```

Comment:
It's probably better practice to use a tag or a layer here so that we don't rely on the player GameObject having a certain name ([Unity documentation on tags and layers](https://docs.unity3d.com/Manual/class-TagManager.html)). This can be left as is. I just wanted to point it out.